builds/Makefile

```
@@ -63,14 +63,20 @@ x86-64-slice.o: x86-64-slice.c
 X86_64_SLICE_BASE=$(abspath $(CURDIR)/x86-64-slice)
 X86_64_SLICE_DYLIB=$(X86_64_SLICE_BASE).dylib
 X86_64_SLICE_A=$(X86_64_SLICE_BASE).a
-X86_64_SLICE:=$(X86_64_SLICE)$(suffix $@)
+X86_64_SLICE_NOEXT=$(X86_64_SLICE_BASE)
+X86_64_SLICE=$(X86_64_SLICE_BASE)$(suffix $@)
+
+X86_64_SLICE_ALL=$(X86_64_SLICE_DYLIB) $(X86_64_SLICE_A) $(X86_64_SLICE_NOEXT)
 
 $(X86_64_SLICE_DYLIB): x86-64-slice.o
 	$(MAC_CC) -shared -o$@ $<
 
 $(X86_64_SLICE_A): x86-64-slice.o
 	$(AR) -rv $@ $<
 
+$(X86_64_SLICE_NOEXT): $(X86_64_SLICE_DYLIB)
+	cp $< $@
```

This makes it silent and fast (CP expands to `cp -c`):

```suggestion
	$(Q) $(CP) $< $@
```

---

of course, thank you 🙂